eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -0,0 +1,538 @@
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Process\PhpExecutableFinder;
+use PDO;
+
+class UpdateTimestampsToUTCCommand extends ContainerAwareCommand
+{
+    const DEFAULT_ITERATION_COUNT = 100;
+
+    /**
+     * @var int
+     */
+    protected $done = 0;
+
+    /**
+     * @var string
+     */
+    protected $timezone;
+
+    /**
+     * @var string
+     */
+    private $mode;
+
+    /**
+     * @var string
+     */
+    private $from;
+
+    /**
+     * @var string
+     */
+    private $to;
+
+    /**
+     * @var \Doctrine\DBAL\Connection
+     */
+    private $connection;
+
+    /**
+     * @var string
+     */
+    private $phpPath;
+
+    /**
+     * @var bool
+     */
+    private $dryRun;
+
+    protected function configure()
+    {
+        $this
+            ->setName('ezplatform:timestamps:to-utc')
+            ->setDescription('Updates ezdate & ezdatetime timestamps to UTC')
+            ->addArgument(
+                'timezone',
+                InputArgument::OPTIONAL,
+                'Original timestamp TimeZone',
+                null
+            )
+            ->addOption(
+                'dry-run',
+                null,
+                InputOption::VALUE_NONE,
+                'Execute a dry run'
+            )
+            ->addOption(
+                'mode',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Select conversion scope: date, datetime, all',
+                'all'
+            )
+            ->addOption(
+                'only-datetime',
+                null,
+                InputOption::VALUE_NONE,
+                'Only ezdatetime fields will be converted'
+            )
+            ->addOption(
+                'from',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Only versions AFTER this date will be converted',
+                null
+            )
+            ->addOption(
+                'to',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Only versions BEFORE this date will be converted',
+                null
+            )
+            ->addOption(
+                'offset',
+                null,
+                InputArgument::OPTIONAL,
+                'Offset for updating records',
+                0
+            )
+            ->addOption(
+                'iteration-count',
+                null,
+                InputArgument::OPTIONAL,
+                'Limit how much records get updated by single process',
+                self::DEFAULT_ITERATION_COUNT
+            )
+            ->setHelp(
+                <<<'EOT'
+The command <info>%command.name%</info> updates field
+data_int in configured Legacy Storage database for a given field type.
+
+Fields will be checked and updated only if not already in UTC timezone.
```

Comment:
will it detect this by itself? :)

Also, as a second sentence here maybe say something like:
`This is to be used either when migrating from eZ Publish to eZ Platform (when using platform backend instead of legacy), or when upgrading legacy to v2019.03 which has been adapted to use UTC.`